src/kaocha/runner.clj

```diff
@@ -1,14 +1,14 @@
 (ns kaocha.runner
   "Main entry point for command line use."
   (:gen-class)
-  (:require [clojure.java.io :as io]
+  (:require [kaocha.api :as api]
+            [clojure.java.io :as io]
```

Comment:
Is this necessary because it's influencing the load order? If so then we should put a big fat comment around it, since I will often let Emacs automatically sort my ns declerations, so this will inadvertently break probably as soon as I touch this namespace.

Reply:
I added a comment. This still seems a bit too fragile to me. What if I created a separate file just for the version check and `require`d  it before the namespace declaration?

Reply:
I think that's a good idea!